docs/DeveloperGuide.md

```diff
@@ -333,7 +339,6 @@ GradPad allows users to edit modules that have already been added.
 
 The following fields of a module can be edited:
 * Module Code
-* Modular Credits
 * Tags
 
 As with all operations in GradPad, the `EditCommand` class handles the execution of edit operations.
```

Comment:
for line 349, I think we can remove the setting of `ModularCredit` for an `EditModuleDescriptor` since we no longer support that

Reply:
Also for line 349-350, we might have to include that at least one field is compulsory, cause now its kinda saying all are optional